### DISCUSSION TITLE
Interface events | deleted/activated/deactivated

### DISCUSSION THREAD

**Discussion by mabicca** [strongswan/strongswan#1039] [Q&A]
Hi everyone,

Just wanted to try and sort these events out:

charon-systemd: interface enP62742s1 deleted
charon-systemd: interface enP62742s1 activated
charon-systemd: interface enP62742s1 deactivated

Are these anything that we should worry about, meaning that strongswan is relying on these interfaces or these are mostly informational events?

These are happening randomly in an environment in a virtual environment where we have the synthetic driver using eth0 and a physical real interface being passed through and binding to eth0, so anytime there is an actual adapter issue, that adapter gets swapped and the data path switched back to the same eth0, all interface parameters in ipsec.conf and what not are set to eth0 since that is the interface that we should be using, but seeing these messages around the times we see problems, seems like there's something else that could be going on since when this happens, some of the ipsec tunnels don't come back gracefully.

Any info is appreciated.

Thank you,
-Marco

**Comment by tobiasbrunner** [comment]
> Are these anything that we should worry about, meaning that strongswan is relying on these interfaces or these are mostly informational events?

The _kernel-netlink_ plugin tracks all interfaces on the system. Whether those messages are relevant depends on what kind of interface that is and whether it's relevant to exchange traffic. The interface used by the IKE daemon is determined by the routes on the system.
